.gitignore

```diff
@@ -1 +1,3 @@
 draft
+go.sum
+*/.idea/
```

Comment:
```suggestion
```

You can add it to your local global .gitignore file, and avoid adding this to each project.